File: algorithm-programmers-binarysearch-immigration/src/test/java/com/currenjin/SolutionTest.java
package com.currenjin;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class SolutionTest {
    @Test
    void sample1() {
        assertEquals(28L, new Solution().solution(6, new int[]{7, 10}));
    }

    @Test
    void oneInspector() {
        assertEquals(50L, new Solution().solution(10, new int[]{5}));
    }
}

File: algorithm-programmers-binarysearch-immigration/src/test/kotlin/com/currenjin/SolutionTest.kt
package com.currenjin

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class SolutionTest {
    @Test
    fun sample1() {
        assertEquals(28L, Solution().solution(6, intArrayOf(7, 10)))
    }

    @Test
    fun oneInspector() {
        assertEquals(50L, Solution().solution(10, intArrayOf(5)))
    }
}
